pages/index.tsx

```diff
@@ -34,7 +35,7 @@ import { fuseSearch } from 'utils'
 
 export const HomePage: NextPage = () => {
   const { data: allResources } = useQuery(['getAllResources'], () =>
-    getAllResources()
+    getAllResources("NOT({status} = 'Do Not Publish')")
```

Comment:
Is this from a boolean marked draft? Or is this string in the Airtable row?

Reply:
This is [filtering](https://support.airtable.com/docs/formula-field-reference#logical-operators-and-functions) on the `status` that already exists on the `Resources` table. I think `Published`/`Draft` is for collections.